test/Examples/PlaneCurve-test.jl

```diff
@@ -346,10 +346,52 @@ end
 @testset "Counting Points on Elliptic Curves" begin
        K = GF(5, 7)[1]
        S, (x, y, z) = PolynomialRing(K, ["x", "y", "z"])
-       T = grade(S)[1]
+       T, _ = grade(S)
        F = T(-x^3 - 3*x^2*y - 3*x*y^2 - x*z^2 - y^3 + y^2*z - y*z^2 - 4*z^3)
        PP = projective_space(K, 2)
        P1 = Oscar.Geometry.ProjSpcElem(PP[1], [K(-1), K(1), K(0)])
        E = Oscar.ProjEllipticCurve(F, P1)
        @test order(E) == 78633
 end
+
+@testset "Torsion points on elliptic curves" begin
+	S, (x, y, z) = PolynomialRing(QQ, ["x", "y", "z"])
+	T, _ = grade(S)
+	F = T(-x^3 - 3*x^2*y - 3*x*y^2 - x*z^2 - y^3 + y^2*z - y*z^2 - 4*z^3)
+	PP = projective_space(QQ, 2)
+	P1 = Oscar.Geometry.ProjSpcElem(PP[1], [QQ(-1), QQ(1), QQ(0)])
+	E = Oscar.ProjEllipticCurve(F, P1)
+	Q1 = Oscar.Point_EllCurve(E, P1)
+	P2 = Oscar.Geometry.ProjSpcElem(PP[1], [QQ(-2), QQ(2), QQ(1)])
+	Q2 = Oscar.Point_EllCurve(E, P2)
+	@test Oscar.order(Q1) == 1
+	@test Oscar.order(Q2) == 0
+	@test istorsion_point(Q1)
+	@test !istorsion_point(Q2)
+	@test Oscar.torsion_points_lutz_nagell(E) == [Q1]
+	@test Oscar.torsion_points_division_poly(E) == [Q1]
+end
+
+@testset "Elliptic Curve Method" begin
+	n = Oscar.ECM(ZZ(4453))
+	@test gcd(n, ZZ(4453)) == n
+
+	A = ResidueRing(ZZ, ZZ(4453))
+	S, (x, y, z) = PolynomialRing(A, ["x", "y", "z"])
+	T, �_ = grade(S)
```

Comment:
Somehow a non-unicode character slipped in which broke the tests, let's see if I can fix it...
```suggestion
	T, _ = grade(S)
```